src/components/Header/Text/Text.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import TxtCSS from './Text.module.css';
+import PropTypes from 'prop-types';
+
+const Text = (props) =>
+     (
```

Comment:
it is redundant

Reply:
As I understood we are talking about merge Text ana Button in one component